conf.yaml

```
@@ -70,9 +70,9 @@ contents_title:     Elastic Stack and Product Documentation
 # The keys don't really matter, but by convention the are the same as the variable.
 variables:
   stackcurrent: &stackcurrent 7.16
-  stacklive: &stacklive [ master, 8.0, 7.16, 6.8 ]
+  stacklive: &stacklive [ master, 8.0, 7.17, 7.16, 6.8 ]
```

Should only one of 7.x be "live"?
```suggestion
  stacklive: &stacklive [ master, 8.0, 7.17, 6.8 ]
```

---

Because 7.16 is still current, I don't think so. I think we remove 7.16 when 7.17 releases.

---

That makes sense, thanks!